collector/src/main/java/com/bitmovin/analytics/data/DeviceInformationProvider.kt

```diff
@@ -4,11 +4,12 @@ import android.content.Context
 import android.os.Build
 import com.bitmovin.analytics.utils.Util
 
-open class DeviceInformationProvider(val context: Context, val userAgent: String) {
+open class DeviceInformationProvider(val context: Context, val userAgent: String, val isTV: Boolean) {
     fun getDeviceInformation(): DeviceInformation {
         return DeviceInformation(
                 manufacturer = Build.MANUFACTURER,
                 model = Build.MODEL,
+                isTV = isTV,
```

Comment:
I think it makes more sense to just use the `context` and extract the info here :)
The responsibility / name of this class is `DeviceInformationProvider`. It's actually a Factory for DeviceInformation so the logic "is this a tv" would totally fit here and would also be easily tested.
Also the benefit would be that the call site in `BitmovinSdkAdapter` doesn't have to change :)

With this you are actually "spreading" the responsibility for the `isTV` to multiple places